app/web/src/components/AssetFuncListPanel.vue

```diff
@@ -87,10 +87,20 @@ const variantSummaries = computed(() => {
   return variant;
 });
 
-const funcsByKind = computed(() => {
+const funcsByKindWithDeprecationFiltering = computed(() => {
   const r = variantSummaries.value
     ? groupBy(variantSummaries.value.funcSummaries ?? [], (f) => f.kind)
     : {};
+
+  // NOTE(nick): filter out deprecated attribute funcs that have become intrinsic funcs.
+  if (r[FuncKind.Attribute]) {
+    r[FuncKind.Attribute] = r[FuncKind.Attribute].filter((f) => {
+      if (f.name === "si:resourcePayloadToValue") return false;
+      else if (f.name === "si:normalizeToArray") return false;
+      return true;
+    });
+  }
```

Comment:
We only want to filter out the old special case funcs at the last possible moment. This just removes them from the user's view, but they still come up in the route, which is acceptable. We only want to remove them from the authoring experience.